src/pages/tokenDrop/Payment.tsx

```diff
@@ -453,12 +454,32 @@ const Payment: FunctionComponent = () => {
                     </Stack>
 
                     <Box mt={ 1 }>
-                      <Typography variant="subtitle1">
-                        { paymentAddress
-                          ? `Send ${purchasePrice} ADA to the ` +
-                            "payment address above."
-                          : "Submit a wallet address to generate a payment address." }
-                      </Typography>
+                      { paymentAddress ? (
+                        <Alert
+                          severity="info"
+                          iconMapping={ {
+                            info: <Info fontSize="inherit" color="info" />,
+                          } }
+                          sx={ {
+                            backgroundColor: theme.colors.grey600,
+                            borderRadius: "6px",
+                            p: "8px 16px",
+                          } }
+                        >
+                          <Typography
+                            color="blue"
+                            fontWeight={ 600 }
+                            variant="subtitle1"
+                          >
+                            Send { purchasePrice } ADA to the payment address
+                            above.
+                          </Typography>
+                        </Alert>
+                      ) : (
+                        <Typography variant="subtitle1">
+                          Submit a wallet address to generate a payment address.
+                        </Typography>
+                      ) }
```

Comment:
Added figma change requested by Tizi for the [info alert figma design](https://www.figma.com/file/Z1yL2dk96S3CBB4CdnUf9E?node-id=1264:1041#372553992)